addOns/websocket/src/main/javahelp/org/zaproxy/zap/extension/websocket/resources/help/contents/pscanrules.html

```diff
@@ -11,5 +11,28 @@ <H1>WebSocket Passive Scan Rules</H1>
         <h2 id="scripts">Scripts</h2>
         Scripts which are included by default in the add-on and they implement the following WebSocket passive scan rules:
 
+        <h3>Base64 Disclosure</h3>
+
+        This script analyzes incoming WebSocket message payload for Base64 strings. The encoded information may include sensitive data which they may not specifically meant for end user consumption. In addition, those encoded payloads may include file signatures known as Magic Bytes. <br>
```

Comment:
I think the `they` could be removed. I'd leave out the magic bytes reference, in itself is not a problem.

Reply:
Fixed f87355442